demos/src/demo.js

```diff
@@ -2,4 +2,17 @@ import './../../main.js';
 
 document.addEventListener('DOMContentLoaded', function() {
 	document.dispatchEvent(new CustomEvent('o.DOMContentLoaded'));
+
+	document.querySelector('[data-o-component="o-cookie-message-close"]').addEventListener('click', showNotification);
```

Comment:
Is this a race condition with https://github.com/Financial-Times/o-cookie-message/blob/f4fcf83f9761ca8054543b7e30ada24a165edd15/src/js/cookieMessage.js#L38 ?